addons/test_mail/tests/test_message_track.py

```diff
@@ -171,6 +172,14 @@ def test_notify_track_groups(self):
         self.assertFalse(msg_emp.get('tracking_values'), "should not have protected tracking values")
         self.assertTrue(msg_sudo.get('tracking_values'), "should have protected tracking values")
 
+        # test editing the record with user not in the group of the field
+        self.record.invalidate_cache()
+        self.record.clear_caches()
+        record_form = Form(self.record.with_user(self.user_employee))
+        record_form.name = 'mm'
```

Comment:
Well, take some time to write a few more characters, like "TestDoNoCrash". A small comment telling it is linked to a regression causing a crash would be welcomed.